.github/workflows/main.yml

```diff
@@ -1,5 +1,9 @@
 name: Claude Code Review
 
+permissions:
```

Comment:
The patch improves security by specifying minimal required permissions instead of 'write-all'. It also switches to using a personal GitHub token, which is generally more secure than the default GITHUB_TOKEN.

``` 
Consider adding 'issues: write' to the permissions if commenting on issues is needed:

permissions:
  contents: read
  pull-requests: write
  issues: write
```